io-engine/src/bin/io-engine-client/v1/pool_cli.rs

```diff
@@ -41,6 +65,16 @@ pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
                 .index(1)
                 .help("Storage pool name"),
         );
+
+    let export = SubCommand::with_name("export")
+        .about("Export storage pool without destroying it")
```

Comment:
Is 'export' some internal operation i.e. developer/test specific? I'm unclear what 'export without destroying' indicates here.

Reply:
Export and import are terms from Solaris/ZFS world. In SPDK they call it "unload" and "load".

When a pool is destroyed ("destruct" in SPDK terms), the disk is marked as clean. One is free to create a new pool on it.
When a pool is exported ("unloaded"), the disk is marked as occupied. One cannot create a new pool on it, only "import" ("load") the existing pool.

In v0, we had "create" and "destroy" to manage all the cases, but it proved to be impractical. In v1, we added "import" and "export".